install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -276,37 +276,3 @@ subjects:
   - kind: ServiceAccount
     name: machine-api-operator
     namespace: openshift-machine-api
-
----
```

Comment:
pinging @sur to acknowledge that this is the correct move.

Reply:
I don't think there's a need to move this roles